recipes/autoconf-archive/all/conanfile.py

```diff
@@ -47,10 +47,13 @@ def package(self):
         with tools.chdir(os.path.join(self._source_subfolder)):
             self._autotools = self._configure_autotools()
             self._autotools.install()
-        tools.mkdir(os.path.join(self.package_folder, "bin").replace("\\", "/"))
-        tools.rename(os.path.join(self.package_folder, "share").replace("\\", "/"), os.path.join(self.package_folder, "bin", "share").replace("\\", "/"))
+
+        tools.mkdir(os.path.join(self.package_folder, "res"))
+        tools.rename(os.path.join(self.package_folder, "share", "aclocal"),
+                     os.path.join(self.package_folder, "res", "aclocal"))
+        tools.rmdir(os.path.join(self.package_folder, "share"))
 
     def package_info(self):
-        aclocal_path = os.path.join(self.package_folder, "bin", "share", "aclocal").replace("\\", "/")
-        self.output.info("Appending ACLOCAL_PATH environment var: {}".format(aclocal_path))
-        self.env_info.ACLOCAL_PATH.append(aclocal_path)
+        aclocal_path = tools.unix_path(os.path.join(self.package_folder, "res", "aclocal"))
+        self.output.info("Appending AUTOMAKE_CONAN_INCLUDES environment var: {}".format(aclocal_path))
+        self.env_info.AUTOMAKE_CONAN_INCLUDES.append(aclocal_path)
```

Comment:
Looks like we're escaping `AUTOMAKE_CONAN_INCLUDES` correctly, but not `ACLOCAL_PATH` on Windows.

Reply:
Ideally, `AUTOMAKE_CONAN_INCLUDES` is not necessary and we use `ACLOCAL_PATH` 100%.

Reply:
Can you clarify? I don't quite understand this. `ACLOCAL_PATH` is needed for the macros to be in the search path. What's the purpose of `AUTOMAKE_CONAN_INCLUDES`? Where can I read more about that?

Reply:
`AUTOMAKE_CONAN_INCLUDES` is added by a conan patch (before I knew the purpose of `ACLOCAL_PATH` myself.
Its functionality is the same as `ACLOCAL_PATH`, but better supported apparently.

Reply:
understood, thanks